packages/payment-detection/test/any/any-to-erc20-proxy-contract.test.ts

```diff
@@ -77,7 +77,28 @@ describe('api/any/conversion-fee-proxy-contract', () => {
     });
   });
 
-  it('can createExtensionsDataForCreation with fee amount and address', async () => {
+  it('can createExtensionsDataForCreation (Goerli)', async () => {
```

Comment:
It might be interesting to factorize it (regarding the previous test) with network as arg